### ISSUE TITLE
Run time for each scenario. (nice to have)

### ISSUE THREAD

**Issue by VankOde** [leinonen/cucumber-html-report#21]
It would be nice to have an option to add to the report the time that each scenario took to run, the date is already given by the JSON. Took a look and didn't find the implementation. 

So kindly requesting it :)

Thanks

**Comment by leinonen** [comment]
Sure thing. Will add it to my backlog :)

**Comment by leinonen** [comment]
I have added it to version 0.3.7. I just display the values as is. I think the values are in nanoseconds (not 100% sure). I can convert to seconds or something later if that is what you want.

**Comment by VankOde** [comment]
Thanks will take a look and give you feedback !

**Comment by leinonen** [comment]
Status?

**Comment by njayasathya** [comment]
Hi leinonen . I have added an this.after method in the hooks file. In the new version of the report , during each scenario ,in the condition it displating it as after and the step name as the scenario name.. But in old version 0.2.9 it does not came. Please check with that.
Thank you for your effort to this html report

**Comment by njayasathya** [comment]
And also please include scenario wise duration .Now only step duration is coming

**Comment by sriram289** [comment]
yes scenario wise duration and the total execution time would be very useful.
